src/ipahealthcheck/ipa/trust.py

```diff
@@ -195,6 +195,66 @@ def check(self):
                                  domain=domain)
 
 
+@registry
+class IPADomainCheck(IPAPlugin):
+    """
+    Check that the IPA domain provider is configured to use ipa
+    """
+    @duration
+    def check(self):
+        if not self.registry.trust_agent:
```

Comment:
I think this test should actually be run on any IPA master, not just trust agents.

Reply:
Sure, dropped the restriction.